### DISCUSSION TITLE
PDSCH POWER

### DISCUSSION THREAD

**Discussion by christsaks** [srsran/srsRAN_Project#543] [General Help]
Hi to everyone!
Trying to modify and looking ways to change the pdsch transmission power i noticed this in the gnb.yaml as option in the csi :

    pwr_ctrl_offset: 0                      # Optional INT (0). Sets the power offset of PDSCH RE to NZP CSI-RS RE in dB. Supported: [-8 - 15].
If I understand well changing this to greater values should increase the PDSCH transmsit power. However, when i test it using ZMQ and GNU i would expect to see some increase in SNR measures at the UE side, which i dont. Maybe I miss sth and this isn't the actual functionality of this or I miss sth in testing it. If this is the case, is there any way which i can modify the pdsch transmission power?
Any hint?
Thanks in advance!

**Comment by pgawlowicz** [comment]
@christsaks Any update on your issue?

**Reply by christsaks** [reply]
Reading the spec again and trying to understand it better I dont think I have understood the functionality of this. Maybe it is  more to inform the UE on what offset to expect between CSI-RS and PDSCH transmit power rather than actually applying it, i don't really know --I need to study this more. My intention was to somehow "play" with the PDSCH transmit power, sth like downlink power control (?). The spec clearly says downlink power control can be applied by it leaves it to vendor implementation if I get it well. Current version of gnb from what I've seen doesnt support such functionality so I was looking for alternatives to play with trasnmit power. Anyways I'm still working on this.

**Reply by safaldhamala** [reply]
hey any updates as of now? I am working on something similar too, and would like to know where PDSCH transmit power is defined
